sdk/textanalytics/Azure.AI.TextAnalytics/src/DocumentHealthcareResult.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text.RegularExpressions;
+using Azure.AI.TextAnalytics.Models;
+
+namespace Azure.AI.TextAnalytics
+{
+    /// <summary>
+    /// DocumentHealthcareEntities.
+    /// </summary>
+    public partial class DocumentHealthcareResult
+    {
+        internal DocumentHealthcareResult(DocumentHealthcareEntitiesInternal documentHealthcareEntities)
+        {
+            Entities = documentHealthcareEntities.Entities;
+            Relations = documentHealthcareEntities.Relations != null ? ResolveHealthcareRelations(documentHealthcareEntities.Entities, documentHealthcareEntities.Relations) : null;
+            Id = documentHealthcareEntities.Id;
+            Warnings = documentHealthcareEntities.Warnings != null ? Transforms.ConvertToWarnings(documentHealthcareEntities.Warnings) : null;
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="DocumentHealthcareResult"/> class.
+        /// </summary>
+        /// <param name="id"></param>
+        /// <param name="textAnalyticsError"></param>
+        public DocumentHealthcareResult(string id, TextAnalyticsError textAnalyticsError)
+        {
+            Id = id;
+            TextAnalyticsError = textAnalyticsError;
+        }
+
+        internal static IReadOnlyList<HealthcareRelation> ResolveHealthcareRelations(IEnumerable<HealthcareEntity> entities, IEnumerable<HealthcareRelationInternal> relations)
+        {
+            List<HealthcareRelation> list = new List<HealthcareRelation>();
+            foreach (HealthcareRelationInternal relation in relations)
+            {
+                list.Add(new HealthcareRelation(relation.RelationType,
+                    relation.Bidirectional,
+                    ResolveHealthcareEntity(entities, relation.Source),
+                    ResolveHealthcareEntity(entities, relation.Target)));
+            }
+
+            return list;
+        }
+
+        /// <summary> Unique, non-empty document identifier. </summary>
+        public string Id { get; }
+        /// <summary> Healthcare entities. </summary>
+        public IReadOnlyList<HealthcareEntity> Entities { get; }
+        /// <summary> Healthcare entity relations. </summary>
+        public IReadOnlyList<HealthcareRelation> Relations { get; }
+        /// <summary> Warnings encountered while processing document. </summary>
+        public IReadOnlyList<TextAnalyticsWarning> Warnings { get; }
+        /// <summary> if showStats=true was specified in the request this field will contain information about the document payload. </summary>
+        public TextDocumentStatistics? Statistics { get; }
```

Comment:
I don't see this assigned anywhere. maybe I missed it, but want to make sure it is assigned